### PR TITLE
FEATURE: Track Neos site and dimension and add site policies

### PR DESCRIPTION
**Pull request by daniellienert**

### CODE REVIEW

File: Migrations/Mysql/Version20210607194130.php [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+final class Version20210607194130 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Add siteName and contentDimensions';
+    }
+
+    public function up(Schema $schema): void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
```
Comment (andrehoffmann30) [inline Migrations/Mysql/Version20210607194130.php:18]: Nitpick Comment can be removed

File: Migrations/Mysql/Version20210607194130.php [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+        $this->addSql('ALTER TABLE punktde_form_persistence_domain_model_formdata ADD sitename VARCHAR(255) NOT NULL, LONGTEXT NOT NULL COMMENT \'(DC2Type:flow_json_array)\'');
+    }
+
+    public function down(Schema $schema): void
+    {
+        // this down() migration is auto-generated, please modify it to your needs
```
Comment (andrehoffmann30) [inline Migrations/Mysql/Version20210607194130.php:25]: Nitpick Comment can be removed

File: Classes/Domain/Repository/FormDataRepository.php [hunk 3]
```diff
@@ -108,17 +108,29 @@ public function findFormDataWithIdentifierContainingData(string $formIdentifier,
...
-        return $queryBuilder->andWhere($queryBuilder->expr()->in('form.siteName', $this->accessibleSites));
+        if (count($this->accessibleSites) > 0) {
+            return $queryBuilder->andWhere($queryBuilder->expr()->in('form.siteName', $this->accessibleSites));
+        }
+
+        return $queryBuilder->andWhere('2 = 1');
+    }
+
+    /**
+     * @return string[]
```
Comment (andrehoffmann30) [inline Classes/Domain/Repository/FormDataRepository.php:119]: This shows the wrong return type.

Comment (daniellienert) [reply Classes/Domain/Repository/FormDataRepository.php:119]: no, it doesn't ? It returns an array of strings.

**Pull request merged**